### PR TITLE
replace sqrtf with algorithms::math::sqrt

### PR DESCRIPTION
**Pull request by PrometheusPi**
This is a pull request replaces `sqrtf` by `algorithms::math::sqrt` in the picongpu main code.
This is similar to #1461 where @Flamefire replaced `sqrtf` in libPMacc.

### CODE REVIEW

**Comment by Flamefire** [comment]
I'd suggest to instantiate the template with `sqrt_x` which was introduced to allow double precision square roots taken from single precision values

File: src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp [hunk 1]
```diff
@@ -129,7 +129,7 @@ namespace picongpu
             return elong;
 #else
             //beam waist in the near field: w_y(y=0) == W0
-            const float_X w_y = W0 * sqrtf( float_X(1.0) + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
+            const float_X w_y = W0 * algorithms::math::sqrt( float_X(1.0) + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
             //! the Gouy phase shift
             const float_X xi_y = atanf( -FOCUS_POS / y_R );
```
Comment (psychocoderHPC) [inline src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp]: Could you please use `math::atan` here

Comment (PrometheusPi) [inline src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp]: @psychocoderHPC Should I know add this in another pull request (as discussed *offline*  3 days ago) or should I add this to this pull request?

Comment (psychocoderHPC) [inline src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp]: Please add a new pull request

**Pull request merged**